tools/scripts/brokenLinkChecker/brokenLinkChecker.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+
+const blc = require('broken-link-checker');
+const https = require('https');
+const url = require('url');
+
+var whitelist = require('./whitelist.json').ignore;
+
+const slack_url = "https://hooks.slack.com/services/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";
+//"https://hooks.slack.com/services/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";
```

Comment:
Currently posting to a private test room.  I'll change this to post to the staging room (the commented-out URL) before merging.